Nef_3/doc/Nef_3/CGAL/boost/graph/convert_nef_polyhedron_to_polygon_mesh.h

```diff
@@ -5,9 +5,48 @@ namespace CGAL {
 /// Note that contrary to `Nef_polyhedron_3::convert_to_polyhedron()`, the output is not triangulated
 /// (but faces with more than one connected component of the boundary).
 /// The polygon mesh can be triangulated by setting `triangulate_all_faces` to `true` or by calling the function `triangulate_faces()`.
+/// @tparam Nef_polyhedron an object of type `Nef_polyhedron_3`.
+/// @tparam Polygon_mesh a model of `MutableFaceGraph` with an internal property map for `CGAL::vertex_point_t`.
+///
+/// The points from `nef` to `pm` are converted using
+/// `CGAL::Cartesian_converter<NefKernel, TargetKernel>`.
+/// `NefKernel` and `TargetKernel` are deduced using `CGAL::Kernel_traits`
+/// from the point type of `nef` and the value type of the vertex_point_map of `tm`.
+/// 
+/// @param nef the input.
+/// @param pm the output.
+/// @param triangulate_all_faces indicates whether all the faces must be triangulated.
+/// 
 /// \pre `Polygon_mesh` must have an internal point property map with value type being `Nef_polyhedron_3::Point_3`.
 /// \pre `nef.simple()`
   
   template <class Nef_polyhedron, class Polygon_mesh>
   void convert_nef_polyhedron_to_polygon_mesh(const Nef_polyhedron& nef, Polygon_mesh& pm, bool triangulate_all_faces = false);
+  
+  /// \ingroup PkgNef3IOFunctions
+  /// Converts an objet of type `Nef_polyhedron_3` into a polygon soup.
+  /// The polygons can be triangulated by setting `triangulate_all_faces` to `true`.
+  /// @tparam Nef_polyhedron an object of type `Nef_polyhedron_3`.
+  /// @tparam PointRange a model of the concepts `RandomAccessContainer` and 
```

Comment:
Check but I don't think `RandomAccessContainer` is even needed.

Reply:
It might not be needed in the function but it wouldn't be very usable, as `polygons` contains ids and not iterators...